modules/develop/partials/run-transforms.adoc

```diff
@@ -38,6 +38,15 @@ rpk profile create --from-profile <(kubectl get configmap --namespace <namespace
 Replace `<profile-name>` with the name that you want to give this `rpk` profile.
 endif::[]
 
+== Limitations
+
+- Transforms have no external access to disk or network resources.
+- Only single record transforms is supported, but multiple output records from a single input record is supported. For aggregations, joins, or complex transformations, use Apache Flink.
+- Only a single output topic is supported.
```

Comment:
This is gone in 24.1

Reply:
Will be removing in a separate PR aimed at 24.1